.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       
       - run: |
           cd ethereum-network-partition-test
-          go test -v .
+          go test -timeout 20m  -v
```

Comment:
20m?? that's ... a lot!

Reply:
The default was ten minutes and the tests were timing out so went with this. I think Pari had made some changes that change fork / genesis stuff

Reply:
Changed it to 15m. Will ask Pari if this jump is expected.